src/test/java/pl/edukacyjni/user/UserServiceTests.java

```diff
@@ -0,0 +1,40 @@
+package pl.edukacyjni.user;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.edukacyjni.pair.Pair;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+public class UserServiceTests {
+    @InjectMocks
+    private UserService userService;
+
+    @Test
+    public void checkEmailValidity_ShouldReturnTrue_WhenValidEmailAddressGiven() {
```

Comment:
It can stay as it is but we need to remember to set up some naming schema for that in real project